build.sbt

```
@@ -2414,10 +2416,13 @@ lazy val `runtime-language-epb` =
         "org.graalvm.sdk"      % "nativeimage" % graalMavenPackagesVersion
       ),
       Compile / internalModuleDependencies := Seq(
+        (`jvm-channel` / Compile / exportedModule).value,
+        (`jvm-interop` / Compile / exportedModule).value,
         (`ydoc-polyfill` / Compile / exportedModule).value,
         (`runtime-utils` / Compile / exportedModule).value
       )
     )
+    .dependsOn(`jvm-interop` % Test)
```

You probably want
```suggestion
    .dependsOn(`jvm-interop` % "test->test")
```
?
The former means that tests are included in regular build.

---

> The former means that tests are included in regular build.

- I don't believe that's the case, but:
- how would I find something is wrong?
- looking at `-cp` value in `.enso-sources-classes` and `.enso-sources-test` seems to be correct:
- e.g. `EpbJvmObjectTest` has `lib/java/jvm-interop/target/scala-2.13/classes` in classpath
- but `EpbLanguage` doesn't

e.g. I believe `.dependsOn(`jvm-interop` % Test)` modifies classpath just for tests of `runtime-epb-language` project.